src/DI/ConsoleExtension.php

```diff
@@ -121,21 +111,20 @@ public function loadConfiguration()
 
 	/**
 	 * Decorate services
-	 *
-	 * @return void
 	 */
-	public function beforeCompile()
+	public function beforeCompile(): void
 	{
 		$builder = $this->getContainerBuilder();
 		$config = $this->validateConfig($this->defaults);
 
 		// Skip if isn't CLI
-		if ($this->cliMode !== TRUE) return;
+		if ($this->cliMode !== true)
```

Comment:
Same above

Reply:
I always accidentally run autoformating and not found yet how to disable that in phpstorm, sorry

Reply:
No problem ;-)